functions/go/format/README.md

```diff
@@ -0,0 +1,73 @@
+# format
+
+### Overview
+
+<!--mdtogo:Short-->
+
+Format the field ordering in resources.
+
+<!--mdtogo-->
+
+### Synopsis
+
+<!--mdtogo:Long-->
+
+The `format` function formats the field ordering in YAML configuration files. Field
+ordering follows the ordering defined in the source Kubernetes resource definitions,
```

Comment:
this still leaves many questions for the user (me in this case). Expand with one or two sentences of how this works:

What's the "source Kubernetes resource definition"? Will this use the OpenAPI document, and if so explicitly state that. Otherwise this is too vague. Presumably, once we have support for injecting OpenAPI document, this function will take advantage of that (you don't need to mention that mechanssm here).

Reply:
Updated.